app-lt-2-10/instance-initializers/fastboot/head.js

```diff
@@ -15,5 +15,9 @@ export function initialize(instance) {
 
 export default {
   name: 'head-fastboot',
-  initialize: initialize
+  initialize() {
+    if (typeof FastBoot !== 'undefined') {
```

Comment:
Why wrap this in a check? Is this to prevent these initializers to run in the browser post FastBoot 1.0?